tritium-metrics/src/main/java/com/palantir/tritium/metrics/MetricRegistries.java

```diff
@@ -318,7 +318,7 @@ public static ScheduledExecutorService instrument(
             String name) {
         return new TaggedMetricsScheduledExecutorService(
                 checkNotNull(delegate, "delegate"),
-                checkNotNull(registry, "registry"),
+                ExecutorMetrics.of(registry),
```

Comment:
should we still check non-null?

Reply:
I didn't retain that null check because our generated utility does the same check internally. No strong preference either way.